SnakeGame/SnakeGame/Utils/ScoreManager.cs

```diff
@@ -0,0 +1,36 @@
+﻿using SnakeGame.Constants;
+using SnakeGame.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SnakeGame.Utils
+{
+    internal class ScoreManager : IScoreManager
+    {
+        public int Score { get; set; } = 0;
+
+        public ScoreManager()
+        {
+        }
+
+        public void Add(int n)
+        {
+            Score += n;
+        }
+
+        public void Render()
+        {
+            Writer.WriteAt(0, 0, $"Score: {Score}");
+        }
+
+        public void RenderLogo()
```

Comment:
We could have something like fancyMessageRenderer which takes the score and do the rendering of all logos. I think score manager should not be responsible for that also